src/Draco.Editor.Web/app/build.js

```diff
@@ -2,6 +2,9 @@ import { build } from 'esbuild';
 import * as path from 'path';
 import * as fs from 'fs';
 import { fileURLToPath } from 'url';
+import { CreateThemeBasedLogo } from './logo-handler.js';
+
+CreateThemeBasedLogo();
```

Comment:
Avoid putting it at the top of the file like this before all the helper method.
Behavior like this one is put at the end of the file.